build-tools/packages/build-infrastructure/package.json

```diff
@@ -0,0 +1,88 @@
+{
+	"name": "@fluid-tools/build-infrastructure",
+	"version": "0.45.0",
+	"private": true,
+	"description": "Fluid build infrastructure",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "build-tools/packages/build-infrastructure"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"type": "module",
+	"exports": {
+		".": {
+			"import": {
+				"types": "./lib/index.d.ts",
+				"default": "./lib/index.js"
+			},
+			"require": {
+				"types": "./dist/index.d.ts",
+				"default": "./dist/index.js"
+			}
+		}
+	},
+	"main": "lib/index.js",
+	"scripts": {
+		"build": "fluid-build . --task build",
+		"build:commonjs": "npm run tsc && npm run build:test",
```

Comment:
We need CJS build because the build-tools package is CJS-only, I assume? And later we should convert it and this to ESM-only?